src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -632,26 +632,26 @@ m_lastInputTime(0)
     // change our window's decorations and functions according to the requested style)
     if (!m_fullscreen)
     {
-        Atom WMHintsAtom = getAtom("_MOTIF_WM_HINTS", false);
-        if (WMHintsAtom)
+        Atom wmHintsAtom = getAtom("_MOTIF_WM_HINTS", false);
+        if (wmHintsAtom)
         {
-            constexpr unsigned long MWM_HINTS_FUNCTIONS   = 1 << 0;
-            constexpr unsigned long MWM_HINTS_DECORATIONS = 1 << 1;
+            constexpr unsigned long mwmHintsFunctions   = 1 << 0;
+            constexpr unsigned long mwmHintsDecorations = 1 << 1;
 
             //constexpr unsigned long MWM_DECOR_ALL         = 1 << 0;
-            constexpr unsigned long MWM_DECOR_BORDER   = 1 << 1;
-            constexpr unsigned long MWM_DECOR_RESIZEH  = 1 << 2;
-            constexpr unsigned long MWM_DECOR_TITLE    = 1 << 3;
-            constexpr unsigned long MWM_DECOR_MENU     = 1 << 4;
-            constexpr unsigned long MWM_DECOR_MINIMIZE = 1 << 5;
-            constexpr unsigned long MWM_DECOR_MAXIMIZE = 1 << 6;
+            constexpr unsigned long mwmDecorBorder   = 1 << 1;
+            constexpr unsigned long mwmDecorResizeh  = 1 << 2;
+            constexpr unsigned long mwmDecorTitle    = 1 << 3;
+            constexpr unsigned long mwmDecorMenu     = 1 << 4;
+            constexpr unsigned long mwmDecorMinimize = 1 << 5;
+            constexpr unsigned long mwmDecorMaximize = 1 << 6;
 
             //constexpr unsigned long MWM_FUNC_ALL          = 1 << 0;
-            constexpr unsigned long MWM_FUNC_RESIZE   = 1 << 1;
-            constexpr unsigned long MWM_FUNC_MOVE     = 1 << 2;
-            constexpr unsigned long MWM_FUNC_MINIMIZE = 1 << 3;
-            constexpr unsigned long MWM_FUNC_MAXIMIZE = 1 << 4;
-            constexpr unsigned long MWM_FUNC_CLOSE    = 1 << 5;
+            constexpr unsigned long mwmFuncResize   = 1 << 1;
+            constexpr unsigned long mwmFuncMove     = 1 << 2;
+            constexpr unsigned long mwmFuncMinimize = 1 << 3;
+            constexpr unsigned long mwmFuncMaximize = 1 << 4;
+            constexpr unsigned long mwmFuncClose    = 1 << 5;
```

Comment:
I think those constant should keep the previous formatting to match the constants of [Motif library](https://sourceforge.net/p/motif/code/ci/master/tree/lib/Xm/MwmUtil.h) from which they come from if I understand correctly.

Reply:
Because these are all defined in succession it's not too hard to wrap them in a `// NOLINTBEGIN` block. I could be convinced to make that change if others also don't want to change these.

Reply:
Agree with kimci86 here; maybe a comment could clarify why there is this exception.

Maybe a bit off-topic, but why don't we directly include that header? Dependency decoupling?

Reply:
No problem. I reverted those changes.